### PR TITLE
stream deploy output via actioncable

### PR DESCRIPTION
**Pull request by grosser**
/cc @zendesk/samson @ragurney 

WIP wil break a bunch of tests but seems to work ... let's go over this tomorrow to see if anything seems obviously broken ...

basically making job output a single stream that all users consume, then request the header as necessary and filter themselves from the viewers

### CODE REVIEW

File: app/views/shared/_output_stream.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% if job.active? %>
```
Comment (ragurney) [inline app/views/shared/_output_stream.html.erb]: So `viewers` tab only works on active jobs. Maybe we hide that tab if job is inactive?

File: app/assets/javascripts/channels/job_outputs_channel.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    viewers: function(users){
+      users = users.filter(function(user) { return user.id != userId});
+      if (users.length > 0) {
+        var viewers = $.map(users, function(user) {
+          return user.name;
+        }).join(', ') + '.';
+
+        $('#viewers-link .badge').html(users.length);
+        $("#viewers").html('Other viewers: ' + viewers);
```
Comment (ragurney) [inline app/assets/javascripts/channels/job_outputs_channel.js]: XSS vulnerability 😱

**Comment by grosser** [comment]
ready!

**Comment by ragurney** [comment]
What about removing the route / other stream-urls?

**Comment by grosser** [comment]
done

**Comment by grosser** [comment]
streaming works everywhere now, so here goes nothing ...

**Pull request merged**